packages/tempus/src/Tempus_IntegratorPseudoTransientForwardSensitivity_impl.hpp

```diff
@@ -171,8 +161,8 @@ void
 IntegratorPseudoTransientForwardSensitivity<Scalar>::
 setTempusParameterList(Teuchos::RCP<Teuchos::ParameterList> pl)
 {
-  state_integrator_->setTempusParameterList(pl);
-  sens_integrator_->setTempusParameterList(pl);
+  TEUCHOS_TEST_FOR_EXCEPTION( true, std::logic_error,
+      "  IntegratorPseudoTransientForwardSensitivity::setTempusParameterList() --  Deprecated!\n");
```

Comment:
Go ahead and mark this function as deprecated with TEMPUS_HIDE_DEPRECATED_CODE, if it is still needed by applications.

Reply:
having the state integrator call the `IntegratorBasic->setTempusParameterList` instead for a clean code.